app/client/src/features/dashboard/DashboardEventListItem.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+import { Button, ListItem, ListItemSecondaryAction } from '@mui/material';
+import { DashboardEvent } from '@local/features/dashboard/DashboardEvent';
+
+interface DashboardEventListItemProps {
+    event: {
+        id: string;
+        title: string | null;
+        description: string | null;
+        startDateTime: Date | null;
+        endDateTime: Date | null;
+        organization: {
+            name: string;
+        } | null;
+    };
+    ongoing: boolean;
+    divider: boolean;
+}
+
+export function DashboardEventListItem({ event, ongoing, divider }: DashboardEventListItemProps) {
+    const router = useRouter();
+    const handleNav = (path: string) => () => router.push(path);
+
+    return (
+        <ListItem key={event.id} divider={divider} button onClick={handleNav(`/events/${event.id}/settings`)}>
+            {event.title && event.description && event.startDateTime && event?.organization?.name && (
```

Comment:
A helper function would make this more clear IMO.

```tsx
/* Dictates whether or not an event is valid to display to the user */
function isValidEvent(event: Event) {
 return event.title && ...
}
// ...
{isEventValid(event) && ...}
```
This can be defined outside of the react function since it's so simple and doesn't really need to be in the render function.